workflows/basic/main.tf

```
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+Create a workflow that reads and stores the current date and then retrieves
+a list of Wikipedia articles related to the day of the week.
+See https://cloud.google.com/workflows/docs/create-workflow-terraform
+before running the code snippet.
+*/
+
+# [START workflows_terraform_api_enable]
+# Enable Workflows API
+resource "google_project_service" "workflows" {
+  service            = "workflows.googleapis.com"
+  disable_on_destroy = false
+}
+# [END workflows_terraform_api_enable]
+
+# [START workflows_terraform_serviceaccount_create]
+# Create a dedicated service account
+resource "google_service_account" "workflows_service_account" {
+  account_id   = "sample-workflows-sa"
+  display_name = "Sample Workflows Service Account"
+}
+# [END workflows_terraform_serviceaccount_create]
+
+# [START workflows_terraform_workflow_deploy]
+# Create a workflow
+resource "google_workflows_workflow" "workflows_example" {
```

Resources defined once should use `default` name. 
```suggestion
resource "google_workflows_workflow" "default" {
```